### DISCUSSION TITLE
PSP Deprecated

### DISCUSSION THREAD

**Discussion by Vertiwell** [topolvm/topolvm#578] [Q&A] [answered]
Is there a recommended config to move off PodSecurityPolicy for 1.25? That I can use with the helm chart?
Something along the lines of:
```
apiVersion: v1
kind: Namespace
metadata:
  name: ${NAMESPACE}
  labels:
    [pod-security.kubernetes.io/enforce](http://pod-security.kubernetes.io/enforce): privileged
    [pod-security.kubernetes.io/audit](http://pod-security.kubernetes.io/audit): privileged
    [pod-security.kubernetes.io/warn](http://pod-security.kubernetes.io/warn): privileged
```
and:
```
podSecurityPolicy:
  create: false
```
In the helm chart.

**Comment by pluser** [comment, marked as answer]
Thank you for your question.

If you want to disable PodSecurityPolicy, that is the correct way to write it.
As for how to set PodSecurityAdmission, we do not know how to write it, so please refer to the Kubernetes documentation.
